### PR TITLE
Fixed #25549 -- Added auth_user_password_change urlreverse example.

### PR DESCRIPTION
**Pull request by darkryder**
[Ticket](https://code.djangoproject.com/ticket/25549)

### CODE REVIEW

File: docs/ref/contrib/admin/index.txt [hunk 1]
```diff
@@ -2806,6 +2806,11 @@ if you specifically wanted the admin view from the admin instance named
     >>> change_url = urlresolvers.reverse('admin:polls_choice_change',
     ...                                   args=(c.id,), current_app='custom')
 
+If you want to change the password of a user, the URL can be resolved by::
```
Comment (timgraham) [inline docs/ref/contrib/admin/index.txt]: I'd rather use the table format as done for the other URLs.

```
The ``UserAdmin`` provides an additional named URL:

======================  ===============================================   =============
Page                    URL name                                          Parameters
======================  ===============================================   =============
Password change         auth_user_password_change                         user_id
======================  ===============================================   =============
```
Okay with you?

File: docs/ref/contrib/admin/index.txt [hunk 2]
```diff
@@ -2806,6 +2806,11 @@ if you specifically wanted the admin view from the admin instance named
...
+
+    >>> urlresolvers.reverse('admin:auth_user_password_change', args=(1,))
+    u'/admin/auth/user/1/password/'
```
Comment (timgraham) [inline docs/ref/contrib/admin/index.txt]: I don't think we need the example, but remember no u'' prefix as our docs are written with Python 3 in mind.

**Comment by darkryder** [comment]
Sure, though I'll fix this tomorrow.
Thanks a ton! :smile:

**Comment by timgraham** [comment]
merged in ad167502f36f2547718aee290d4d11bf7a26ff82, thanks!

**Pull request closed without merging**